### ISSUE TITLE
[Feature Request] Set Playlist to private or public upon creation

### ISSUE THREAD

**Issue by meleemaster101** [jmshrv/finamp#879]
The Jellyfin web UI has the option to set a playlist as public to the server or private to the user upon creation. Could we add this to the create playlist dialogue in finamp?

**Comment by Chaphasilor** [comment]
Do you necessarily need to choose between the options, or would a default option be enough for now?  
I could add a default pretty easily, but choosing the type during creation will take a bit longer...

**Comment by jar-113** [comment]
Either option would work. For my use, I don't see a huge need for the option for choosing during creation. Just a default option would be amazing!

**Comment by Chaphasilor** [comment]
Okay. Just to make sure, what is the current behavior? Does it create public playlists by default?

**Comment by jar-113** [comment]
Yes. So when someone else makes a playlist, it show up for me in my Finamp app and vice versa. When I create the playlist first within Jellyfin, it prompts me if I want it to be a publlic playlist or not. The Finapp app just always makes public playlists as far as I can tell.

Jellyfin Server: 10.10.1
Finamp App: 0.9.12

**Comment by Chaphasilor** [comment]
This is now possible since version 0.9.14 :)

**Comment by jar-113** [comment]
Awesome! I see it! Thanks!
